packages/help-center/src/route-to-query-mapping.json

```diff
@@ -0,0 +1,83 @@
+{
+	"/home/": "home",
+	"/stats/": "statistics",
```

Comment:
Replace current suggestions with: 

https://wordpress.com/support/stats/
https://wordpress.com/support/stats/understand-your-sites-traffic/
https://wordpress.com/support/stats/learn-insights-about-your-website/
https://wordpress.com/support/google-analytics/
https://wordpress.com/support/getting-more-views-and-traffic/